org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/SwitchPatternTest.java

```diff
@@ -3752,32 +3752,30 @@ public void testBug575051_2() {
 				},
 				"");
 	}
-	// Test we do report illegal fall-through to pattern
```

Comment:
@mpalat I don't see any explanation why this test was removed. Accident? Or could you explain?